knowledge-base/actions/damccorm/tag-ur-it/action-security.yml

```diff
@@ -0,0 +1,12 @@
+name: "tag-ur-it"
+github-token:
+  action-input:
+    input: GITHUB_TOKEN
+    is-default: false
+  permissions:
```

Comment:
I think there is only `issues: write` permission here, view [here](https://github.com/damccorm/tag-ur-it/blob/7c0e852a6bd84e9c7f842ef4f0ee67335b005a49/src/index.ts#L79).